package.json

```diff
@@ -13,6 +13,7 @@
     "minor": "./release.sh minor",
     "patch": "./release.sh patch",
     "start": "cd demo && npm run start",
+    "dev": "cd demo && npm install && npm run dev",
```

Comment:
I ran into this (needing `npm install` in `/demo` the other day on a new machine). If we add that to the `/demo` package's `npm run start` command, is that enough to not need these `npm run demo` commands? I'm pretty neutral, but asking because they look largely similar besides the `npm install` and `open`